lib/menu.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'order'
+require 'yaml'
+
+class Menu
+
+  MENU = YAML::load(File.open('./lib/resources/menu.yml'))
+
+  def initialize(order = Order)
+    @order_klass = order
+    @order = @order_klass.new
+  end
+
+  def view
+    output = "---- MENU -----\n"+readable_list
+  end
+
+  def select(dish_name, quantity = 1)
+    dish = find(dish_name)
+    dish = w_quantity(find(dish_name), quantity)
+    @order.add(dish)
+  end
+
+  def order
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)